routes/login.js

```diff
@@ -35,7 +35,7 @@ router.post('/signup', function(req, res, next) {
 })
 
 router.put('/signin', urlencodedParser, async(req, res) => {
-    console.log(req.body)
+    console.log(req.body);
```

Comment:
later in the future, you will need to clean up every console.log so that it actually helps us to figure out what is going on with the program